sbin/functions.sh

```diff
@@ -966,6 +966,189 @@ clean_up()
+				;;
+		esac
+	else
+		# default to current dir
+		ARTIFACTS_DIR="`pwd`"
+	fi
+	echo "ARTIFACTS_DIR=\"${ARTIFACTS_DIR}\"" >> ${localconf}
+	filecount=`ls $ARTIFACTS_DIR -1 | grep -v menifest | wc -l`
+	if [ $filecount -lt 3 ]; then
+		debugmsg ${DEBUG_CRIT} "Artifacts are not sufficient. Please verify."
+		return 1
+	fi
+
+	# exclude manifest files
+	filelist=`ls $ARTIFACTS_DIR | grep -v manifest` menuitems=""
+	local checklistmenu=""
+	for i in $filelist; do
+		menuitems="$menuitems $i $i "
+		checklistmenu="$checklistmenu $i $i off "
+	done
+
+	# Kernel selection
+	dialog --no-tags --menu "Please choose kernel:" \
+		$(expr $filecount + 7) 100 $filecount $menuitems 2>$tmpfile
+	dialog_res=$(cat $tmpfile)
+	if [ -z "${dialog_res}" ]; then
+		debugmsg ${DEBUG_CRIT} "Kernel not specified. Please try again."
+		return 1
+	fi
+	echo "INSTALL_KERNEL=\"\${ARTIFACTS_DIR}/${dialog_res}\"" >> ${localconf}
+
+	# Rootfs selection
+	dialog --no-tags --menu "Please choose rootfs:" \
+		$(expr $filecount + 7) 100 $filecount $menuitems 2>$tmpfile
+	dialog_res=$(cat $tmpfile)
+	if [ -z "${dialog_res}" ]; then
+		debugmsg ${DEBUG_CRIT} "Rootfs not specified. Please try again."
+		return 1
+	fi
+	echo "INSTALL_ROOTFS=\"\${ARTIFACTS_DIR}/${dialog_res}\"" >> ${localconf}
+	echo "HDINSTALL_ROOTFS=\"\${ARTIFACTS_DIR}/${dialog_res}\"" >> ${localconf}
+
+	# Initramfs selection
+	dialog --no-tags --menu "Please choose initramfs:" \
+		$(expr $filecount + 7) 100 $filecount $menuitems 2>$tmpfile
+	dialog_res=$(cat $tmpfile)
+	if [ -z "${dialog_res}" ]; then
+		debugmsg ${DEBUG_CRIT} "Initramfs not specified. Please try again."
+		return 1
+	fi
+	echo "INSTALL_INITRAMFS=\"\${ARTIFACTS_DIR}/${dialog_res}\"" >> ${localconf}
+
+	# containers selection
+	dialog --no-tags --checklist "Please choose install containers:" \
+		$(expr $filecount + 7) 100 $filecount $checklistmenu 2>$tmpfile
+	dialog_res=$(cat $tmpfile)
+	if [ -z "${dialog_res}" ]; then
+		debugmsg ${DEBUG_CRIT} "Containers not specified. Please try again."
+		return 1
+	fi
+
+	local hdcontainers=${dialog_res}
+
+	# network prime selection
+	menuitems=""
+	filecount=0
+	for i in ${hdcontainers}; do
+		menuitems="$menuitems $i $i "
+		let filecount++
+	done
+	dialog --no-tags --menu "Please choose the network prime container:" \
+		$(expr $filecount + 7) 100 $filecount $menuitems 2>$tmpfile
+	local network_prime_container=$(cat $tmpfile)
+	if [ -z "${network_prime_container}" ]; then
+		debugmsg ${DEBUG_INFO} "Network prime containers not specified."
+	fi
+
+	# Format HDINSTALL_CONTAINERS configuration
+	echo "HDINSTALL_CONTAINERS=\" \\" >> ${localconf}
+	for i in ${hdcontainers}; do
+		echo -n " \${ARTIFACTS_DIR}/${i}" >> ${localconf}
+		if [ "${i}" == "${network_prime_container}" ]; then
+			echo -n ":net=1" >> ${localconf}
+		fi
+		echo " \\" >> ${localconf}
+	done
+	echo " \"" >> ${localconf}
+
+	# Basic configures
+    cat <<EOF >> ${localconf}
+EVAL_NAME="Nucleo-T Evaluation"
```

Comment:
random comment, we should update this to OverC .. and if there are other nucleo-t references, squash them as well.